EliteDangerous/JournalEvents/JournalScan.cs

```diff
@@ -487,13 +487,25 @@ public string DisplayString(int indent = 0, bool includefront = true)
             return scanText.ToNullSafeString().Replace("\n", "\n" + inds);
         }
 
+        public string GetHabZoneStringLs()
+        {
+            if (IsStar && HabitableZoneInner.HasValue && HabitableZoneOuter.HasValue)
+            {
+                return $"{HabitableZoneInner.Value.ToString("N0")}-{HabitableZoneOuter.Value.ToString("N0")}ls";
```

Comment:
The C# 6.0 interpolated string (which is typically handled by [`FormattableString`](https://msdn.microsoft.com/en-us/library/system.formattablestring(v=vs.110).aspx)) is a vast improvement over the olden days, but there's no need to use `{object.ToString("fmt")}` inside of one. Simply `{object:fmt}` is all it needs. In other words:
```CS
return $"{HabitableZoneInner.Value.ToString("N0")}-{HabitableZoneOuter.Value.ToString("N0")}ls";
```
is syntactically identical to
```CS
return $"{HabitableZoneInner.Value:N0}-{HabitableZoneOuter.Value:N0}ls";
```

Reply:
To further clarify, if either of the `double?` types are null, the two code blocks are not actually identical. `string s = $"{HabitableZoneInner.Value}";` will NRE if `HabitableZoneInner` is null, whereas `string s = $"{HabitableZoneInner}";` will yield an empty string.

Reply:
Combination of trying to make the minimal change, and coding before 0600... good catch.

Both methods are protecting against null `double?`s already so I can't see the problem there.

Reply:
It's not a problem since the `null` checks are already there. I just wanted to correct my misstatement of "is syntactically identical to", to be "is *almost* syntactically identical to (unless `null` is involved)."